files/newrelic.ini

```diff
@@ -0,0 +1,858 @@
+; This file contains the various settings for the New Relic PHP agent. There
+; are many options, all of which are described in detail at the following URL:
+; https://docs.newrelic.com/docs/agents/php-agent/configuration/php-agent-configuration
+;
+
+; If you use a full path to the extension you insulate yourself from the
+; extension directory changing if you change PHP installations or versions.
+; If you do not use an absolute path then the file must be installed in the
+; active configuration's extension directory.
+extension = "newrelic.so"
+
+[newrelic]
+; Setting: newrelic.enabled
+; Type   : boolean
+; Scope  : per-directory
+; Default: true
+; Info   : Enable or disable the agent. Please note that you cannot globally
+;          disable the agent and then selectively enable it on a per-directory
+;          basis. If you disable the agent in the global INI file then the
+;          agent will not initialize at all. However, you can selectively
+;          disable the agent on a per-directory basis.
+;
+newrelic.enabled = "${NEW_RELIC_ENABLED}"
```

Comment:
Not clear to me how we are going to get a value in here. ini files aren't evaluated before loading, and not modifyable once running.

Reply:
It'll get environment variables from the pod. See https://github.com/universityofadelaide/shepherd/pull/162

Reply:
You can test this locally as well by manually building this image and using it as the FROM in the local Dockerfiler then adding the environment variables in the web pod.

Reply:
https://www.php.net/manual/en/configuration.file.php#example-36 - So good!